lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/utils/LambdaCloudDriverUtils.java

```diff
@@ -205,12 +205,7 @@ private Headers buildHeaders() {
         return headersBuilder.build();
     }
 
-    public boolean lambdaExists(LambdaGetInput inp) {
-        LambdaDefinition thisLambda = retrieveLambda(inp);
-        return thisLambda != null;
-    }
```

Comment:
I'd probably keep this for future use. looks like a good helper function even if not used currently.

Reply:
I'd rather keep the code clean, add it back as needed since it is merely a wrapper around the retrieveLambda code.  That's a personal view... if I left it I'd probably look at renaming it to "lambdaExistsInCache" since that's what it was doing.